iFixit/react-commerce
# Track GA4 Events

## Overview

We want to track GA4 events in react-commerce now, so this pull gets GA4's `gtag()` working consistently and also adds the event tracking functionality we need. 

We will now track events
`view_item`
`view_cart`
`view_item_list`
`add_to_cart`
`remove_from_cart`

I don't think any other events need to be covered in `react-commerce`

## QA
Make sure that the above events are successfully tracked in the app. I've personally found 
https://chrome.google.com/webstore/detail/analytics-debugger/ilnpmccnfdjdjjikgkefkcegefikecdc/related helpful for seeing the events, but you should probably check the live debugger.

Connect: https://github.com/iFixit/ifixit/issues/49333
